static/js/textbox.html

```diff
@@ -12,6 +12,36 @@
             return vars;
         }
 
+        function disableNavButtons(){
+            var next_buttons = window.parent.document.getElementsByClassName('button-next');
+            var previous_buttons = window.parent.document.getElementsByClassName('button-previous');
+            var i;
+            for (i = 0; i < next_buttons.length; i ++){
+                next_buttons[i].classList.add('disabled');
+                next_buttons[i].setAttribute('disabled', 'disabled');
+                previous_buttons[i].classList.add('disabled');
+                previous_buttons[i].setAttribute('disabled', 'disabled');
+            }
+        }
+
+        function enableNextButtons(problemLink) {
+            var next_buttons = window.parent.document.getElementsByClassName('sequence-nav-button button-next');
+            var i;
+            console.log(problemLink);
+            for (i = 0; i < next_buttons.length; i ++){
+                next_buttons[i].classList.remove('disabled');
+                next_buttons[i].removeAttribute('disabled', 'disabled');
+                var clone = next_buttons[i].cloneNode();
```

Comment:
there is no other way to remove or disable Event listeners registered earlier than our one